src/main/java/org/assertj/core/condition/MappedCondition.java

```diff
@@ -131,13 +135,33 @@ public boolean matches(FROM value) {
    * @return the mapped condition description .
    */
   protected String buildMappingDescription(FROM from, TO to) {
+  	return buildMappingDescription(from,to,true);
+  }
+
+  private String buildMappingDescription(FROM from, TO to,boolean withNested) {
     StringBuilder sb = new StringBuilder("mapped");
     if (!mappingDescription.isEmpty()) sb.append(format("%n   using: %s", mappingDescription));
     sb.append(format("%n   from: <%s> %s%n", from.getClass().getSimpleName(), from));
     sb.append(format("   to:   <%s> %s%n", to.getClass().getSimpleName(), from, to));
     sb.append("   then checked:");
-    sb.append(format("%n      %-10s", condition));
+    if(withNested) {
+    	sb.append(format("%n      %-10s", condition));
+    }
     return sb.toString();
   }
 
+  @Override
+  public Description conditionDescriptionWithStatus(FROM actual) {
+  	TO mappedObject = mapping.apply(actual);
+    Description descriptionsWithStatus =  condition.conditionDescriptionWithStatus(mappedObject);
+   String status= status(actual).label;
```

Comment:
no need to call `label` as it is returned by `Status.toString()`

Reply:
Condition uses also 
```
  public Description conditionDescriptionWithStatus(T actual) {
    Status status = status(actual);
    return new TextDescription(status.label + " " + description().value());
  }
  ```
  
  and there is no StingBuilder Construnctor for an Object

Reply:
true but in the code above we should have used `status + " " + description().value()`